components/listmarkersmap/markerlist.htm

```diff
@@ -0,0 +1,42 @@
+<div class="marker-group">
+    {% for groupSlug,markers in groups %}
+    <h3>{{ markers.first.group.name ?? 'initbiz.leafletpro::lang.components.list_markers_map.ungrouped'|trans }}</h3>
+    <ul>
+        {% for marker in markers %}
+        <li>
+            <a onclick="centerTo({{ marker.lat }}, {{ marker.lon }})" href="javascript:void(0)">{{ marker.name }}</a>
```

Comment:
This will break if we have more than one map on a page. Don't use the `window` object for this purpose, please. I would prefer to make it fully locally, from `onclick`, or fully globally using a predefined class.